<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IRemoteWindow</class>
 <widget class="QMainWindow" name="IRemoteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IRemoteWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Interface</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0" rowspan="2">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Profile:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="profileCombo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="profileAddButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="list-add">
              <normaloff/>
             </iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="profileRemoveButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="list-remove">
              <normaloff/>
             </iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <widget class="QTableWidget" name="tableWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <column>
           <property name="text">
            <string>Button</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Command</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="addButton">
            <property name="text">
             <string>&amp;Add</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+A</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeButton">
            <property name="text">
             <string>&amp;Remove</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+R</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="imageButton">
            <property name="text">
             <string>Load Image...</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+L</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="editableCheck">
          <property name="text">
           <string>Editable</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+R</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Testing">
       <attribute name="title">
        <string>Testing</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Network</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Hostname</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="networkAddressEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>169.254.1.1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="networkConnectButton">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Port</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" rowspan="2">
            <widget class="QSpinBox" name="networkPortSpin">
             <property name="maximum">
              <number>65000</number>
             </property>
             <property name="value">
              <number>2000</number>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QFrame" name="networkStatusFrame">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: red;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="lineWidth">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>SerialPort</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1">
            <widget class="QPushButton" name="serialConnectButton">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QComboBox" name="serialDeviceCombo">
             <property name="currentIndex">
              <number>1</number>
             </property>
             <item>
              <property name="text">
               <string>ttyACM0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ttyACM1</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QFrame" name="serialStatusFrame">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: red;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="lineWidth">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Commands</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="2" column="0">
            <widget class="QPushButton" name="runButton">
             <property name="text">
              <string>Run</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QPushButton" name="captureButton">
             <property name="text">
              <string>Capture</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="5">
            <widget class="QListWidget" name="commandList"/>
           </item>
           <item row="2" column="3" colspan="2">
            <widget class="QPushButton" name="removeCommandButton">
             <property name="text">
              <string>Remove</string>
             </property>
             <property name="icon">
              <iconset theme="list-remove">
               <normaloff/>
              </iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Testing</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="2">
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>Settings</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0" colspan="2">
               <widget class="QGroupBox" name="wlanGroupBox">
                <property name="title">
                 <string>WLAN</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <layout class="QGridLayout" name="gridLayout_7">
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>SSID</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1" colspan="2">
                  <widget class="QLineEdit" name="wlanSsidEdit"/>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Security</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Passphrase</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1" colspan="2">
                  <widget class="QLineEdit" name="wlanPassphraseEdit">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1" colspan="2">
                  <widget class="QComboBox" name="wlanSecurityCombo">
                   <property name="currentIndex">
                    <number>0</number>
                   </property>
                   <item>
                    <property name="text">
                     <string>Open</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>WEP-128</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>WPA1</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Mixed WPA1 &amp; WPA2-PSK</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>WPA2-PSK</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Adhoc</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>WPE-64</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="4" column="2">
                  <widget class="QPushButton" name="wlanScanButton">
                   <property name="text">
                    <string>Scan...</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Hostname</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1" colspan="2">
                  <widget class="QLineEdit" name="wlanHostnameEdit">
                   <property name="text">
                    <string>IRemoteBox</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="QPushButton" name="wlanAdhocButton">
                   <property name="text">
                    <string>Adhoc</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="3" column="0" colspan="2">
               <widget class="QGroupBox" name="groupBox_7">
                <property name="title">
                 <string>IR</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_8">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Repeat:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QSpinBox" name="irRepeatSpin">
                   <property name="suffix">
                    <string> x </string>
                   </property>
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="value">
                    <number>5</number>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_13">
                   <property name="text">
                    <string>Timeout:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QSpinBox" name="irTimeoutSpin">
                   <property name="suffix">
                    <string> ms</string>
                   </property>
                   <property name="minimum">
                    <number>5</number>
                   </property>
                   <property name="maximum">
                    <number>500</number>
                   </property>
                   <property name="value">
                    <number>50</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QPushButton" name="settingsReadButton">
                <property name="text">
                 <string>Read</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QPushButton" name="settingsSubmitButton">
                <property name="text">
                 <string>Submit</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="2" column="0" colspan="2">
               <widget class="QGroupBox" name="groupBox_6">
                <property name="title">
                 <string>IPv4 Settings</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_11">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Method:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>IP-Address:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QComboBox" name="ipMethodCombo">
                   <property name="currentIndex">
                    <number>0</number>
                   </property>
                   <item>
                    <property name="text">
                     <string>Automatic (DHCP)</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Manual</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Auto-IP (Adhoc)</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLineEdit" name="subnetMaskEdit">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>255.255.0.0</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="ipAddressEdit">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>169.254.1.1</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QLineEdit" name="gatewayEdit">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>Subnet-Mask:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="label_12">
                   <property name="text">
                    <string>Gateway:</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
